README.md

```diff
@@ -1,3 +1,517 @@
+```
+
+#### `display` (`optional`, `Android only`)
+
+Defines the visual display of the picker for Android and will be ignored for iOS.
+
+List of possible values:
+- `"default"`
+- `"spinner"`
+- `"calendar"` (only for `date` mode)
+- `"clock"` (only for `time` mode)
+
+```js
+<RNDateTimePicker display="spinner" } />
+```
+
+#### `onChange` (`optional`)
+
+Date change handler.
+
+This is called when the user changes the date or time in the UI. It receives the event and the date as parameters.
+
+```js
+setDate = (event, date) => {}
+
+<RNDateTimePicker onChange={this.setDate} />
+```
+
+#### `value` (`required`)
+
+Defines the date or time value used in the component.
+
+```js
+<RNDateTimePicker value={new Date()} />
+```
+
+#### `maximumDate` (`optional`)
+
+Defines the maximum date that can be selected.
+
+```js
+<RNDateTimePicker maximumDate={new Date()} />
+```
+
+#### `minimumDate` (`optional`)
+
+Defines the minimum date that can be selected.
+
+```js
+<RNDateTimePicker minimumDate={new Date()} />
+```
+
+#### `timeZoneOffsetInMinutes` (`optional`, `iOS only`)
+
+Allows to change the timeZone of the date picker, by default it uses the device's time zone.
+
+```js
+// GMT+1
+<RNDateTimePicker timeZoneOffsetInMinutes={60} />
+```
+
+#### `locale` (`optional`, `iOS only`)
+
+Allows to change the locale of the component, by default it uses the device's locale.
+
+```js
+<RNDateTimePicker locale="es-ES" />
+```
+
+#### `is24Hour` (`optional`, `Android only`)
+
+Allows to set the time picker to 24hour format.
+
+```js
+<RNDateTimePicker is24Hour={true} />
+```
+
+#### `minuteInterval` (`optional`, `iOS only`)
+
+The interval at which minutes can be selected.
+Possible values are: `1, 2, 3, 4, 5, 6, 10, 12, 15, 20, 30`
+
+```js
+<RNDateTimePicker minuteInterval={10} />
+```
+## Migration from the older components
+
+RNDateTimePicker is the new common name used to represent the old versions of iOS and Android.
+
+### DatePickerIOS
+
+-  `initialDate` is deprecated, use `value` instead.
+  
+    ```js
+    // Before
+    <DatePickerIOS initialValue={new Date()} />
+    ```
+
+    ```js
+    // Now
+    <RNDateTimePicker value={new Date()} />
+    ```
+
+- `date` is deprecated, use `value` instead.
+
+    ```js
+    // Before
+    <DatePickerIOS date={new Date()} />
+    ```
+
+    ```js
+    // Now
+    <RNDateTimePicker value={new Date()} />
+    ```
+
+- `onChange` now returns also the date.
+  
+    ```js
+    // Before
+    onChange = (event) => {}
+    <DatePickerIOS onChange={this.onChange} />
+    ```
+
+    ```js
+    // Now
+    onChange = (event, date) => {}
+    <RNDateTimePicker onChange={this.onChange} />
+    ```
+
+- `onDateChange` is deprecated, use `onChange` instead.
+  
+    ```js
+    // Before
+    setDate = (date) => {}
+    <DatePickerIOS onDateChange={this.setDate} />
+    ```
+
+    ```js
+    // Now
+    setDate = (event, date) => {}
+    <RNDateTimePicker onChange={this.setDate} />
+    ```
+
+### DatePickerAndroid
```

Comment:
Important callout here is that open picker modals will be updated with the changed prop `value` if a HOC holding state changes it. Previously the component use to close the modal and render a new one on consecutive calls.